Objects/longobject.c

```diff
@@ -3505,21 +3494,13 @@ _PyLong_Frexp(PyLongObject *a, int64_t *e)
     /* Rescale;  make correction if result is 1.0. */
     dx /= 4.0 * EXP2_DBL_MANT_DIG;
     if (dx == 1.0) {
-        if (a_bits == INT64_MAX)
-            goto overflow;
+        assert(a_bits < UINT64_MAX);
```

Comment:
`uint64_t a_bits;` cannot be greater than UINT64_MAX.

Reply:
Yes, but it can be equal.